Tensile/KernelWriterAssembly.py

```diff
@@ -3033,6 +3033,30 @@ def allocateResources(self, kernel):
     kStr += inst("v_readfirstlane_b32", sgpr("WaveId"), vgpr(tmpVgpr), "WaveId")
     self.vgprPool.checkIn(tmpVgpr)
 
+    # Check alpha == 0
+    if self.do["ApplyAlpha"]:
```

Comment:
It's possible for a floating point value to be -0, in which case the comparison will fail.  I don't see a scalar floating point comparison instruction so I guess we either have to use a vector instruction or do some bit manipulation.

Reply:
Found a way to do this without needing extra vector registers. Done!